File: 1semana/Carro.cpp
#include <iostream>
#include <string>
using namespace std;
class Carro{

public:
    string modelo;
    int ano;

    void ligar(){
    cout << "Carro ligado "
        <<modelo<< " fabricado em "
        <<ano<<"";
    }

};
int main(){

    Carro c;
    c.modelo = "Gol";
    c.ano = 2023;
    c.ligar();


return 0;
}

File: 1semana/Carro.java
public class Carro {
    // Atributos da classe
    public String modelo;
    public int ano;

    // Método para dizer "Olá"
    public void dizerOla() {
        System.out.println("Oii, meu carro e " 
        + modelo + " e fabricado em " + ano);
    }
    public static void main(String[] args) {
        // Instância da classe Pessoa
        Carro c = new Carro();

        // Atribuição de valores
        c.modelo = "Gol";
        c.ano = 2023;

        // Chamada do método para dizer "Olá"
        c.dizerOla();
    }
}